src/Parser/ShadowPriest/Modules/Spells/Mindbender.js

```diff
@@ -0,0 +1,51 @@
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+
+import makeWclUrl from 'common/makeWclUrl';
+
+class Mindbender extends Module {
+  _damageDone = 0;
+  _mindbenders = {};
+
+  on_initialized() {
+    this.active = true;
+    this.load();
+  }
+
+  get mindbendersAsArray(){
+    return Object.keys(this._mindbenders).map(key => this._mindbenders[key]);
+  }
+
+  get damageDone(){
+    return this._damageDone;
+  }
+
+  on_byPlayer_summon(event) {
+    const spellId = event.ability.guid;
+    if (spellId === SPELLS.MINDBENDER.id) {
+
+      this._mindbenders[event.timestamp] = {
+        start: event.timestamp,
+        end: event.timestamp + 15000 + (1500 * this.owner.selectedCombatant.traitsBySpellId[SPELLS.FIENDING_DARK_TRAIT.id]),
```

Comment:
Please do not use any *magic numbers* in the code and define them as a constant at the top of your file if they're only going to be used in one place and you don't need any additional info.